docs/pages/developer-guide.md

```diff
@@ -64,28 +64,18 @@ They store the desired configuration of a service and all relevant information r
 
 ## Creating a new standalone framework
 
-The `mesosphere/dcos-commons:latest` Docker image is able to set up a development environment for creating a new standalone framework.
+The `mesosphere/dcos-commons:latest` Docker image contains a development environment capable of creating new standalone frameworks.
 
-In order to set up this environment run the following command:
+In order to set up this environment run the following commands:
 ```bash
-$ FRAMEWORK_ROOT=/home/elezar/frameworks
+$ docker pull mesosphere/dcos-commons:latest # make sure you have the latest image
+$ export FRAMEWORK_ROOT=$HOME/frameworks
 $ docker run --rm -ti \
     -v ${FRAMEWORK_ROOT}:${FRAMEWORK_ROOT} \
         mesosphere/dcos-commons:latest \
-            init ${FRAMEWORK_ROOT} --create-framework pyframework
+            init ${FRAMEWORK_ROOT} --create-framework myframework
 ```
-This will create a folder `/home/elezar/frameworks/myframework` (for example) containing a framework based on the DC/OS SDK template framework. The rest of thes instructions are relative to this folder.
-
-## Creating a new framework in the DC/OS SDK repository
```

Comment:
Note, this docker image doesn't yet support the same operations as the `new-framework.sh` script which is why I have not yet removed this section.

(although it doesn't work yet ... )

Reply:
What operations?

Reply:
(operations is a strong word. It doesn't do the same thing as new-framework.sh)